en/docs/visual-programming/sequence-diagram-view.md

```diff
@@ -79,12 +111,21 @@ function execute() returns error? {
     http:Response response = check reportService->post("/report", result);
 }
 ```
-![graphical screenshot](../img/client-code-representations.png)
 
-## Develop programs in the visual mode
+**Graphical view**
+
+<img src="../../img/client-code-representations.png" alt="graphical screenshot" width="450">
```

Comment:
Incorrect image path.

Reply:
This will work in the docs. Shall we fix the github repo issue through another PR?